ql-dsp-plugin/ql-dsp-pm.h

```diff
@@ -0,0 +1,1871 @@
+struct ql_dsp_pm {
```

Comment:
I presume that this file was generated by Yosys' pattern matcher generator (pmgen). It is better to store the source .pmg file and make the generation part of the plugin build process.

Reply:
I agree, I've added the generation script to generate it during plugin build.